### PR TITLE
Add MentaLiST-0.1.4

### PR DESCRIPTION
**Pull request by dfornika**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/mentalist/0.1.4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/WGS-TB/MentaLiST/archive/v{{ version }}.tar.gz
+  sha256: 7dc6bd983156a28d59ad480bea1d7761ebdcd25087db73ef9ce3fb0e59cd224b
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/mentalist/0.1.4/meta.yaml]: should start with 0

File: recipes/mentalist/0.1.4/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  build:
+    - julia 0.5.2
+    - hdf5 {{ CONDA_HDF5 }}*
+    - mpfr
+    - libxml2
+  run:
+    - julia 0.5.2
+    - hdf5 {{ CONDA_HDF5 }}*
+    - mpfr
+    - libxml2
```
Comment (bgruening) [inline recipes/mentalist/0.1.4/meta.yaml]: Should be pinned to - libxml2 2.9.*

**Comment by dfornika** [comment]
Can I request that this be merged? Thanks.

**Comment by dfornika** [comment]
Is there anything preventing this recipe from being merged?

**Comment by daler** [comment]
@dfornika thank you, looks good and sorry for the delay!

**Pull request merged**